packages/snap/snap.manifest.json

```diff
@@ -7,7 +7,7 @@
     "url": "https://github.com/MetaMask/template-snap-monorepo.git"
   },
   "source": {
-    "shasum": "ZdgdkzvE+5EZJRVUeQhiZhuEvvJUJLfSHvph9ZLvB4w=",
+    "shasum": "dq/9docWCwiIu604ezD+C2M887/OpTTyghLGlfQol7k=",
```

Comment:
nit; will this keep changing on each commit 🤔

Reply:
I believe it changes anytime we edit the src code